Unable to pretty print 3D Cartesian Transformation: Aff_transformationC3 has no member named 'print'

## Issue Details

When attempting to output an `Aff_transformation_3` out to a stream (say `std::cout`) where its mode has been set to "pretty" (using `CGAL::set_pretty_mode`), an error message is spewed out during compilation. Expected result would be similar to that of outputting an instance of `Aff_transformation_2`.

## Source Code

```cpp
#include <iostream>

#include <CGAL/Exact_predicates_inexact_constructions_kernel.h>
#include <CGAL/IO/io.h>

using K = CGAL::Epick;
using T3 = K::Aff_transformation_3;

int main() {
  T3 t(CGAL::IDENTITY);
  CGAL::set_pretty_mode(std::cout);
  std::cout << t << std::endl;
  return 0;
}
```

Compilation output:

```
In file included from /usr/local/include/CGAL/Cartesian/Cartesian_base.h:55,
                 from /usr/local/include/CGAL/Simple_cartesian.h:20,
                 from /usr/local/include/CGAL/Exact_predicates_inexact_constructions_kernel.h:20,
                 from test.cpp:3:
/usr/local/include/CGAL/Cartesian/Aff_transformation_3.h: In instantiation of ‘std::ostream& CGAL::operator<<(std::ostream&, const CGAL::Aff_transformationC3<R>&) [with R = CGAL::Epick; std::ostream = std::basic_ostream<char>]’:
/usr/local/include/CGAL/Aff_transformation_3.h:87:13:   required from ‘std::ostream& CGAL::operator<<(std::ostream&, const CGAL::Aff_transformation_3<R>&) [with R = CGAL::Epick; std::ostream = std::basic_ostream<char>]’
test.cpp:12:16:   required from here
/usr/local/include/CGAL/Cartesian/Aff_transformation_3.h:196:7: error: ‘const class CGAL::Aff_transformationC3<CGAL::Epick>’ has no member named ‘print’
  196 |     t.print(os);
      |     ~~^~~~~
```

## Environment

* Operating system: Linux 5.6.8-arch1-1 x86_64
* Compiler: g++ version 9.3.0
* Release or debug mode: Release
* Specific flags used (if any): Just linker flags (`-lboost -lCGAL`)
* CGAL version: 5.0.2
* Boost version: 1.72.0
* Other libraries versions if used (Eigen, TBB, etc.): N/A

--- a/Cartesian_kernel/include/CGAL/Cartesian/Aff_transformation_3.h
+++ b/Cartesian_kernel/include/CGAL/Cartesian/Aff_transformation_3.h
@@ -178,6 +178,9 @@ public:
   Aff_transformation_3 operator*(const Aff_transformationC3 &t) const
   { return (*this->Ptr()) * (*t.Ptr()); }
 
+  std::ostream &
+  print(std::ostream &os) const;
+
   bool operator==(const Aff_transformationC3 &t)const
   {
     for(int i=0; i<3; ++i)
@@ -197,13 +200,21 @@ protected:
 };
 
 
+template < class R >
+std::ostream&
+Aff_transformationC3<R>::print(std::ostream &os) const
+{
+  this->Ptr()->print(os);
+  return os;
+}
+
 #ifndef CGAL_NO_OSTREAM_INSERT_AFF_TRANSFORMATIONC3
 template < class R >
-std::ostream &operator<<(std::ostream &os,
-                         const Aff_transformationC3<R> &t)
+std::ostream&
+operator<<(std::ostream &os, const Aff_transformationC3<R> &t)
 {
-    t.print(os);
-    return os;
+  t.print(os);
+  return os;
 }
 #endif // CGAL_NO_OSTREAM_INSERT_AFF_TRANSFORMATIONC3
 

--- a/Cartesian_kernel/include/CGAL/Cartesian/Aff_transformation_rep_3.h
+++ b/Cartesian_kernel/include/CGAL/Cartesian/Aff_transformation_rep_3.h
@@ -192,8 +192,8 @@ public:
   virtual std::ostream &print(std::ostream &os) const
   {
     os <<"Aff_transformationC3("<<t11<<' '<<t12<<' '<<t13<<' '<<t14<<std::endl;
-    os <<"                    "<< t21<<' '<<t22<<' '<<t23<<' '<<t24<<std::endl;
-    os <<"                    "<< t31<<' '<<t32<<' '<<t33<<' '<<t34<<")";
+    os <<"                     "<<t21<<' '<<t22<<' '<<t23<<' '<<t24<<std::endl;
+    os <<"                     "<<t31<<' '<<t32<<' '<<t33<<' '<<t34<<")";
     return os;
   }
 

--- a/Homogeneous_kernel/include/CGAL/Homogeneous/Aff_transformationH2.h
+++ b/Homogeneous_kernel/include/CGAL/Homogeneous/Aff_transformationH2.h
@@ -783,6 +783,17 @@ operator*(const Aff_transformationH2<R>& right_argument) const
 }
 
 template <class R>
+std::ostream&
+operator<<(std::ostream& out, const Aff_transformationH2<R>& t)
+{
+  typename R::RT RT0(0);
+  Aff_transformation_repH2<R> r = t.Ptr()->general_form();
+  return out << "| "<< r.a << ' ' << r.b << ' ' << r.c << " |\n"
+             << "| "<< r.d << ' ' << r.e << ' ' << r.f << " |\n"
+             << "| "<< RT0 << ' ' << RT0 << ' ' << r.g << " |\n";
+}
+
+template <class R>
 Aff_transformationH2<R>
 _general_transformation_composition( Aff_transformation_repH2<R> l,
                                      Aff_transformation_repH2<R> r )

